content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,153 @@
+---
+reviewers:
+title: Security Checklist
+description: >
+  Recommended checklist for ensuring security in Kubernetes clusters.
+content_type: concept
+---
+
+<!-- overview -->
+
+This checklist aims at providing a basic list of guidance with links to more
+comprehensive documentation on each topic. It does not claim to be exhaustive
+and is meant to evolve.
+
+The order of topics does not reflect an order of priority.
+
+<!-- body -->
+
+## Authentication & Authorization
+
+See the [Role Based Access Control Good Practices](/docs/concepts/security/rbac-good-practices/)
+for information related to authentication and authorization.
+
+<!-- New authentication certificates should be generated on new clusters that avoid -->
+<!-- the use of the `system:masters`. `system:masters` should be used as a -->
+<!-- break-glass mechanism as opposed to an admin user. -->
+
+## Network security
+
+A number of Container Network Interfaces (CNIs) provide functionality to
+restrict network resources pods may communicate with. This is most commonly done
+through [Network Policies](/docs/concepts/services-networking/network-policies/)
+which provide a namespaced resource to define rules.
+
+Not all CNIs provide encryption in transit. If the chosen CNI lacks this
+feature, an alternative solution could be to use a service mesh to provide that
+functionality.
+
+The `etcd` datastore of the control plane should have controls to limit access.
+Furthermore, mutual TLS (mTLS) should be used to communicate securely with it.
+The certificate authority for this should be unique to `etcd`.
+In addition, external Internet access to the Kubernetes API server should be restricted
+to not expose the API publicly. You can then use a bastion host to access the server.
+If a cloud provider is used for hosting Kubernetes, the access from pods to the cloud
+metadata API `169.254.169.254` should also be restricted or blocked if not needed
+because it may leak information.
+
+## Pod security
+
+RBAC authorization is crucial but
+[cannot be specific enough on the Pods resource](/docs/concepts/security/rbac-good-practices/#workload-creation)
+(or on any resource that manages Pods). The only granularity is the API verbs
+on the resource itself, for example `create` on Pods. Indeed, without
+additional admission, the authorization to create these resources allows direct
+unrestricted access to on the schedulable nodes of a cluster.
+
+[Pod Security Admission](/docs/concepts/security/pod-security-admission/) is an
+admission controller to enable fine-grained authorization of Pod creation and
+updates. It replaces the deprecated (as v1.21)
+[Pod Security Policy](/docs/concepts/policy/pod-security-policy/) admission
+controller. The [Pod Security Standard](/docs/concepts/security/pod-security-standards/)
```

Comment:
```suggestion
updates. It replaces the deprecated (as v1.21)
[Pod Security Policy](/docs/concepts/policy/pod-security-policy/) admission
controller. The [Pod Security Standard](/docs/concepts/security/pod-security-standards/)
```
The Pod Security Policy page might be removed when 1.25 is released

Reply:
We could just remove the link and just reference PSP by name? That way at least people who know PSP can read that its replaced and what by

Reply:
```suggestion
updates. It replaces the Pod Security Policy admission controller. 
[Pod Security Standards](/docs/concepts/security/pod-security-standards/)
```